### ISSUE TITLE
Unknown object caused collision in ROAR easy map

### ISSUE THREAD

**Issue by RyanC1681** [augcog/ROAR#51]
**Describe the bug**
There is unknown object caused collision at a corner in ROAR easy map, but it didn't report collision.
Could you check the ROAR easy map?

https://user-images.githubusercontent.com/40802485/121737038-33137c00-caad-11eb-8749-3ef593c92718.mp4

Thanks,
**To Reproduce**
1. Go to 'roar_easy_windows'
2. Click on 'CarlaUE4.exe' to launch
3. Open terminal to run "python .\runner_competition_evaluator.py"
4. See attached video the car spin over near the corner in ROAR easy map, but it didn't report collision on the left.

**Expected behavior**
Expect the car can run near the corner in ROAR easy map without spin over or report collision.

**Screenshots**
Attached the video

**Desktop (please complete the following information):**
 - OS: Win10

**Comment by wuxiaohua1011** [comment]
Hi, could you kindly inform me the following:
1. which agent are you using: ex https://github.com/augcog/ROAR/blob/main/runner_competition_evaluator.py#L82
2. what is your PC specs: ex: Intel i7, Nvidia GTX 1080
3. Did you make any revision to the code? especially in ROAR_Sim? 

Best,
Michael Wu

**Comment by RyanC1681** [comment]
Hi, here is the info as below:
which agent are you using: agent_class = PIDAgent
what is your PC specs: tried Intel i7-10875H, Nvidia T1000 laptop and Desktop PC  Intel i7 with GTX 1060 but car failed at the same spot as attached video.
Did you make any revision to the code? especially in ROAR_Sim?  I tried to increase target speed to 100, but it still failed even I revised back to default=80.  I found the car will spin over when it drove into the shade area in right lane at the corner.  The car can pass if it drove toward left lane in brighter area.

   Is it possible to have remote server to test the ROAR competition to make sure it doesn't depend on the PC hardware? 
Thanks,

https://user-images.githubusercontent.com/40802485/121759095-f493b680-cad8-11eb-9fd2-1dc8b145f9e0.mp4

https://user-images.githubusercontent.com/40802485/121759187-594f1100-cad9-11eb-9936-74a3cad490c4.mp4
